src/ol/render/canvas/Immediate.js

```diff
@@ -83,6 +84,14 @@ class CanvasImmediateRenderer extends VectorContext {
      */
     this.transform_ = transform;
 
+    /**
+     * @private
+     * @type {number}
+     */
+    this.transformRotation_ = transform
```

Comment:
Something looks wrong here. `transform` is a transform matrix, not a number.

Reply:
The rotation is the number.  I first check that the transform is not undefined or null (which is should never be) as several browser tests which create a CanvasImmediateRenderer without using `drawImages_` or `drawText_` fail if they attempt to access entries from a missing matrix.